gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Thu Aug 25 07:56:28 PDT 2016
+#Sun Feb 26 21:35:32 PST 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-3.0-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-3.0-all.zip
```

Comment:
I thought about that, but wasn't sure if you wanted to stick with 3.0 to make sure pygradle doesn't start requiring a later version of gradle...

For debugging, any version with the `-all` flavor will do.